gamemode/modules/afk/cl_afk.lua

```diff
@@ -1,4 +1,4 @@
-local TextColor = Color(GetConVarNumber("Healthforeground1"), GetConVarNumber("Healthforeground2"), GetConVarNumber("Healthforeground3"), GetConVarNumber("Healthforeground4"))
+local TextColor = Color(GetConVar("Healthforeground1"):GetFloat(), GetConVar("Healthforeground2"):GetFloat(), GetConVar("Healthforeground3"):GetFloat(), GetConVar("Healthforeground4"):GetFloat())
```

Comment:
I didn't check and based it on experience with RGB in other places (eg. Android uses integers).